Library/Formula/pgroonga.rb

```diff
@@ -0,0 +1,39 @@
+class Pgroonga < Formula
+  desc "PostgreSQL plugin to use Groonga as index"
+  homepage "https://pgroonga.github.io/"
+  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
+  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
+
+  depends_on "pkg-config" => :build
+  depends_on "groonga"
+  depends_on "postgresql"
+
+  def install
+    # To find HOMEBREW_PREFIX/bin/pg_config
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
```

Comment:
Still think bypassing superenv on the bin is going to cause a bunch of problems for end users, but if you're happy to deal with those if they come up I won't poke you about it yet again.

Sent from my iPhone

> On 4 Nov 2015, at 06:32, Xu Cheng <notifications@github.com> wrote:
> 
> In Library/Formula/pgroonga.rb:
> 
> > @@ -0,0 +1,39 @@
> > +class Pgroonga < Formula
> > +  desc "PostgreSQL plugin to use Groonga as index"
> > +  homepage "https://pgroonga.github.io/"
> > +  url "http://packages.groonga.org/source/pgroonga/pgroonga-1.0.0.tar.gz"
> > +  sha256 "854d66bdc79e7cfceb29ae8706556a3ad1c8b32ac4535939d296966e3b19fd3e"
> > +
> > +  depends_on "pkg-config" => :build
> > +  depends_on "groonga"
> > +  depends_on "postgresql"
> > +
> > +  def install
> > +    # To find HOMEBREW_PREFIX/bin/pg_config
> > +    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
> This also applies to nginx. Perhaps we're better to just bundle the most popular plugins (e.g. those already in core and stuff like postgis) and move the rest to a tap.
> 
> While, nginx doesn't have any problem on prefix sandbox. Currently we have two choice:
> 
> Bundle everything in one formula
> Like this PR, hacking using hard link so plugins can be as individual formulae.
> So should I merge this PR?
> 
> —
> Reply to this email directly or view it on GitHub.
>